src/lifetracker/logic/StorageAdapterImpl.java

```diff
@@ -0,0 +1,28 @@
+package lifetracker.logic;
+
+import java.io.IOException;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import lifetracker.calendar.CalendarList;
+import lifetracker.calendar.CalendarListImpl;
+import lifetracker.storage.Storage;
+
+public class StorageAdapterImpl implements StorageAdapter {
+
+    @Override
+    public void store(CalendarList calendar, Storage storage) throws IOException {
```

Comment:
Instead of passing the storage in each time, just create the Adpater itself with storage in the constructor.